docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.2'
+services:
+  gerrit:
+    image: openfrontier/gerrit:2.15.x
+    container_name: gerrit
+    environment:
+      WEBURL                       : http://gerrit.example.com
+      HTTPD_LISTENURL              : proxy-http://0.0.0.0:8080
+      GITHTTPURL                   : http://git-mirror.example.com
+      SSHD_ADVERTISE_ADDR          : git.example.com:29418
+      SSHD_ENABLE_COMPRESSION      : 'true'
+      DATABASE_TYPE                : postgresql
+      DB_PORT_5432_TCP_ADDR        : postgresdb
+      DB_PORT_5432_TCP_PORT        : '5432'
+      DB_ENV_POSTGRES_DB           : reviewdb
+      DB_ENV_POSTGRES_USER         : gerrit2
+      DB_ENV_POSTGRES_PASSWORD     : gerrit
+      GERRIT_INIT_ARGS             : --batch --install-all-plugins
+      GITWEB_TYPE                  : gitiles
+      USER_NAME                    : gerrit
+      USER_EMAIL                   : gerrit@example.com
+      AUTH_TYPE                    : OAUTH
+      OAUTH_GOOGLE_RESTRICT_DOMAIN : example.com
+      OAUTH_GOOGLE_CLIENT_ID       : ${GOOGLE_CLIENT_ID:-test}
+      OAUTH_GOOGLE_CLIENT_SECRET   : ${GOOGLE_CLIENT_SECRET:-test}
+      MAKE_FIRST_USER_ADMIN        : 'true'
+      ENABLE_SIGNED_PUSH           : 'true'
+      DOWNLOAD_SCHEME              : ssh
+      REPLICATION_REMOTES          : bitbucket
+      BITBUCKET_REMOTE             : https://${BB_USER}@bitbucket.org/${BB_ORG}/${name}.git
+      BITBUCKET_PASSWORD           : ${BITBUCKET_PASSWORD}
+      BITBUCKET_MIRROR             : 'true'
+      BITBUCKET_PROJECTS           : example
+      BITBUCKET_REPLICATE_ON_STARTUP: 'true'
+    ports:
+    - "8080:8080"
+    - "29418:29418"
+    links:
+    - postgresdb
+    depends_on:
+    - postgresdb
+    restart: unless-stopped
+  postgresdb:
+    image: postgres:alpine
+    expose:
+    - 5432
+    environment:
+      POSTGRES_USER: gerrit2
+      POSTGRES_PASSWORD: gerrit
+      POSTGRES_DB: reviewdb
```

Comment:
In general, I'd like to keep this project as simple as possible. That means this project is only about building a docker container. docker-compose or k8s-deployments should be separated to other projects.

Reply:
I understand, but it serves more for documentation on how to run **this** image. Maybe an `example` folder would more intuitive?